Add a new public key

doniset141 opened the pull request:
pub   rsa1024/477471F51EACF860 2021-07-29 [SC]
      8B0A9E02258B25F095A17E0C477471F51EACF860
uid                 [ultimate] Doni Setiawan <ecxalty@gnuweeb.org>
sub   rsa1024/BB6BFC6FD82C3416 2021-07-29 [E]

Signed-off-by: Doni Setiawan <ecxalty@gnuweeb.org>

Commit:
Add a new public key

pub   rsa1024/477471F51EACF860 2021-07-29 [SC]
      8B0A9E02258B25F095A17E0C477471F51EACF860
uid                 [ultimate] Doni Setiawan <ecxalty@gnuweeb.org>
sub   rsa1024/BB6BFC6FD82C3416 2021-07-29 [E]

Signed-off-by: Doni Setiawan <ecxalty@gnuweeb.org>

ammarfaizi2 reviewed (changes requested):
Hey, you just submitted an empty file. Please fix.

Commit:
Add a new public key
pub   rsa1024/477471F51EACF860 2021-07-29 [SC]
      8B0A9E02258B25F095A17E0C477471F51EACF860
uid                 [ultimate] Doni Setiawan <ecxalty@gnuweeb.org>
sub   rsa1024/BB6BFC6FD82C3416 2021-07-29 [E]

Signed-off-by: Doni Setiawan <ecxalty@gnuweeb.org>

Commit:
Add a new public key

pub   rsa1024/477471F51EACF860 2021-07-29 [SC]
      8B0A9E02258B25F095A17E0C477471F51EACF860
uid                 [ultimate] Doni Setiawan <ecxalty@gnuweeb.org>
sub   rsa1024/BB6BFC6FD82C3416 2021-07-29 [E]

Signed-off-by: Doni Setiawan <ecxalty@gnuweeb.org>

ammarfaizi2 reviewed (approved):
Good, I am merging this.
